composer.json

```diff
@@ -1,12 +1,17 @@
 {
     "name": "friendsofsymfony/message-bundle",
     "type": "symfony-bundle",
-    "description": "Provides user-to-user messaging features for your Symfony2 application.",
+    "description": "Provides user-to-user messaging features for your Symfony application.",
     "keywords": ["symfony", "message", "messaging"],
     "homepage": "https://github.com/FriendsOfSymfony/FOSMessageBundle",
     "license": "MIT",
     "authors": [
         {
+            "name": "Titouan Galopin",
```

Comment:
People should open an issue if they have an issue :)
Maybe it should be clearer in the readme but it's imo the best thing to do when having an issue (the maintainer might be busy, don't know, etc... and it's always better to share solutions with the community).

BTW it would be great to have a consistent approach in the symfony communauty for everything touching to authorship `@author` annotation, authors section in composer, etc.